src/app/amopis/ramingen/page.tsx

```diff
@@ -26,14 +26,20 @@ function Estimation() {
           <Breadcrumb.Link href="#">Dashboard</Breadcrumb.Link>
           <Breadcrumb.Link href="#">Planning</Breadcrumb.Link>
         </Breadcrumb>
-        <Heading className="ams-mb--sm">Capaciteitsraming</Heading>
-        <Heading level={2}>Brug 423 opwaardering (Berlagebrug) – ams20192715</Heading>
+        <div className="amopis-visual-headings-order">
```

Comment:
We can get rid of this class name I think

Reply:
Ah, I updated one instance but there were two. Done.